README.md

```diff
@@ -22,6 +22,7 @@ It offers a variety of features as follows:
 ----------
 
 ## Learn it & Contact us
+* Home: [https://rocketmq.apache.org](https://rocketmq.apache.org)
```

Comment:
It could be better to use <> instead of []() , as we don't need to change the display discription here.